ports/azmq/portfile.cmake

```
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO zeromq/azmq
-    REF 6bb101eecb357ad9735ebc36e276b7526652d42d # commit on 2019-05-01 
-    SHA512 18812fd73c3c57aca8b17c2df0af01afb7864253d6ac6ce042e01ef04e81dd454438c9d01f5bb0fd7a4a445082a2eb3dd334ca1e05dafbe4ba602cad95ce7134
+    REF 7da2fd0a1b2bd4e6f50ccd17d54579e6084ef1f7 # commit on 2021-11-20
```

```suggestion
    REF 7da2fd0a1b2bd4e6f50ccd17d54579e6084ef1f7 # v1.0.3
```
Update this annotation to avoid confusion. :)

---

Right, that does make more sense for that comment. Done.